lightning/src/ln/channelmanager.rs

```diff
@@ -6468,7 +6490,7 @@ where
 				if let hash_map::Entry::Occupied(mut chan_phase_entry) = peer_state.channel_by_id.entry(chan_id) {
 					if let ChannelPhase::Funded(chan) = chan_phase_entry.get_mut() {
 						let counterparty_node_id = chan.context.get_counterparty_node_id();
-						let logger = WithChannelContext::from(&self.logger, &chan.context);
+						let logger = WithChannelContext::from(&self.logger, &chan.context, None);
```

Comment:
Technically we do have the `payment_preimage` in this method, which means we could calculate the payment hash. Not sure if worth it but we could. Similar in `internal_update_fulfill_htlc`.